src/packing_queue/tables/HistoryTable.jsx

```diff
@@ -325,38 +325,40 @@ const HistoryTable = ({
     }, 0);
   };
 
-  const onDownloadPDFClick = useCallback(async () => {
-    await API.downloadPDF(
-      selectedRow._id,
-      selectedRow.orderNumber,
-      selectedRow.dateCreatedValue
-    )
-      .then((data) => {
-        pdfMake.createPdf(data.docDefinition).open();
-        setMenuPosition(null);
-        enqueueSnackbar("Packing slip downloaded", snackbarVariants.error);
-      })
-      .catch((e) => {
-        enqueueSnackbar(e.message, snackbarVariants.error);
-      });
-  }, [selectedRow, enqueueSnackbar]);
+  //No longer used mm - 081522
```

Comment:
Any objections to removing this rather than commenting it out?

Reply:
Nope, I removed.